bundler/lib/bundler/dependency.rb

```diff
@@ -7,7 +7,7 @@
 module Bundler
   class Dependency < Gem::Dependency
     attr_reader :autorequire
-    attr_reader :groups, :platforms, :gemfile, :git, :branch
+    attr_reader :groups, :platforms, :gemfile, :git, :branch, :autorequire
```

Comment:
This is already defined in the line above, so it's causing method redefinition warnings that make some tests fail.